src/components/Dashboard/Form/FieldWrapper/FieldWrapper.js

```diff
@@ -1,27 +1,28 @@
 import React from "react";
 import { Field } from "react-final-form";
-import appContent from "../../../../appContent";
+// import appContent from "../../../../appContent";
 import Error from "./ErrorField/ErrorField";
+import styles from "./FieldWrapper.scss";
 import PropTypes from "prop-types";
 
 const required = value => (value ? undefined : "Required");
 
 const FieldWrapper = ({ name, componentSize, component, label }) => {
-  const data = appContent;
+  // const data = appContent;
 
   return (
-    <div className={componentSize}>
-      <label>{data[name]}</label>
+    <div className={styles[componentSize]}>
```

Comment:
In other places, you use a reference to attribute by a dot. Please be consistent.